tests/node/rpc/messages/OnVehicleDataTests.js

```diff
@@ -67,11 +86,13 @@ describe('OnVehicleDataTests', function () {
         const testHandsOffSteering = rpcMessage.getHandsOffSteering();
         const testWindowStatus = rpcMessage.getWindowStatus();
         const testGearStatus = rpcMessage.getGearStatus();
+        const testClimateData = rpcMessage.getClimateData();
 
         // Valid Tests
         Validator.assertEquals(this.stabilityControlsStatus, testStabilityControlsStatus);
         Validator.assertEquals([Test.GENERAL_WINDOW_STATUS], testWindowStatus);
         Validator.assertEquals(this.gearStatus, testGearStatus);
+        Validator.assertEquals(this.climateData, testClimateData);
```

Comment:
Assuming you have setup `Test.js` with the `CLIMATE_DATA`, you can change line 95 to `Validator.assertEquals([TEST.GENERAL_CLIMATE_DATA], testClimateData);`.

Reply:
Fixed in 3881d93